U/Uno/build_tarballs.jl

```diff
@@ -20,10 +20,10 @@ cd build
 
 if [[ "${target}" == *mingw* ]]; then
     LBT=blastrampoline-5
-    LIBHIGHS=${prefix}/lib/libhighs.dll.a
+    HIGHS_DIR=${prefix}/lib
```

Comment:
@cvanaret It seems to be wrong, can you check the logs to see if it is correctly detected on Windows?

Reply:
I'll have a look. That's what we have in Uno's CI script.

Reply:
You did this modification and not me :)

Reply:
I like to live dangerously :) It does look ok though.